test/fixtures/server-test/webpack.array.config.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
+const path = require('path');
+
 module.exports = [{
   mode: 'development',
   context: __dirname,
   entry: './foo.js',
   output: {
     filename: 'foo.js',
-    path: '/js1',
+    path: path.resolve(__dirname, 'js1'),
```

Comment:
curious: previous tests pass on this with the `path.resolve`. is there a reason for this change?

Reply:
As [documented](https://webpack.js.org/configuration/), `output.path` should be an absolute path. And without `path.resolve`, it will attempt to write files into the root directory `/`. This configuration didn't cause an error in the past because files are all in memory when no test cases with `writeToDisk: true`are using it.

Reply:
excellent, thank you